app/account/forms.py

```diff
@@ -112,3 +114,32 @@ class ChangeEmailForm(Form):
     def validate_email(self, field):
         if User.query.filter_by(email=field.data).first():
             raise ValidationError('Email already registered.')
+
+
+class EditProfileForm(Form):
+    first_name = StringField('First name', validators=[
+        DataRequired(),
+        Length(1, 64)
+    ])
+    last_name = StringField('Last name', validators=[
+        DataRequired(),
+        Length(1, 64)
+    ])
+    affiliation_options = [
+        ('1', 'Veteran'),
+        ('2', 'Active Duty'),
+        ('3', 'National Guard'),
+        ('4', 'Reservist'),
+        ('5', 'Spouse'),
+        ('6', 'Dependent'),
+        ('7', 'Family Member'),
+        ('7', 'Supporter'),
+        ('8', 'Other')]
```

Comment:
I think for now I'll leave this in on the front-end. We should definitely not leave it hardcoded in the end - I added a task on Trello to create an Affiliation Model. This way admins can change the options available for Affiliations, and we can display stats on how many users belong to each Affiliation. Maybe once we get the Tags idea more finalized, we can convert this to a Tag? I think that this will probably end up needing different functionality than tags for resources, so maybe we should keep them separate.